src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -95,7 +108,7 @@ class ProjectStacktraceLinkEndpoint(ProjectEndpoint):
     users can go from the file in the stack trace to the
     provider of their choice.
 
-    `filepath`: The file path from the stack trace
+    `file`: The file path from the stack trace
```

Comment:
We do not use `filepath`.

See `https://sentry.io/api/0/projects/sentry/sentry/stacktrace-link/?absPath=%2Fusr%2Fsrc%2Fsentry%2Fsrc%2Fsentry%2Fintegrations%2Fgithub%2Fclient.py&commitId=5d0b682af5c1b71a8ae5b88f7e3ed090264ed1b5&file=sentry%2Fintegrations%2Fgithub%2Fclient.py&module=sentry.integrations.github.client&platform=python&sdkName=sentry.python.django`